tortoise/tests/test_bad_relation_reference.py

```diff
@@ -0,0 +1,94 @@
+from tortoise import Tortoise
+from tortoise.contrib import test
+from tortoise.exceptions import ConfigurationError
+
+
+class TestBadReleationReferenceErrors(test.SimpleTestCase):
+    async def setUp(self):
+        try:
+            Tortoise.apps = {}
+            Tortoise._connections = {}
+            Tortoise._inited = False
+        except ConfigurationError:
+            pass
+        Tortoise._inited = False
+
+    async def tearDown(self):
+        await Tortoise.close_connections()
+        await Tortoise._reset_apps()
+
+    async def test_wrong_app_init(self):
+        with self.assertRaises(ConfigurationError):
```

Comment:
Could you do a `self.assertRaisesRegec(ConfigurationError, "<important substring of the message>"):` here?
It's useful to ensure that the message is correct for the test case.